tox.ini

```diff
@@ -10,7 +10,6 @@ deps =
     -rrequirements.txt
     -rtest-requirements.txt
 commands =
-    black --check
```

Comment:
I fixed black in #97

Reply:
This is just to get the CI pass (https://github.com/hluk/product-listings-manager/actions/runs/2461282276). I will fix it after you merge your PR.